proj/hps_accel/src/tensorflow/lite/kernels/internal/reference/integer_ops/pooling_accel.cc

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 The CFU-Playground Authors
+ * Copyright 2019 The TensorFlow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "tensorflow/lite/kernels/internal/reference/integer_ops/pooling_accel.h"
+
+#include <algorithm>
+#include <cstdio>
+
+#include "cfu.h"
+#include "tensorflow/lite/kernels/internal/common.h"
+
+#define cfu_pool(val0, val1) cfu_op2(0, val0, val1)
```

Comment:
You had already implemented this cfu op for something else?

Reply:
Just added INS_POOL in eb791ec7084c3b0614ba1c36886b53e4c8040fcf. 

I tossed up whether to add more gateware since we're already pushing the boundaries, but the LUT/performance tradeoff seems worth it.

Reply:
Hmm, don't know how I missed that!